src/test/java/org/jenkinsci/plugins/envinject/EnvInjectJobPropertyTest.java

```diff
@@ -0,0 +1,134 @@
+package org.jenkinsci.plugins.envinject;
+
+import hudson.model.Cause;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.ParametersAction;
+import hudson.model.Result;
+import hudson.model.StringParameterValue;
+import hudson.model.TaskListener;
+import hudson.model.queue.QueueTaskFuture;
+import hudson.tasks.Shell;
+import org.junit.Rule;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.junit.Assert.*;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.jvnet.hudson.test.CaptureEnvironmentBuilder;
+
+/**
+ * Tests for {@link EnvInjectJobProperty}
+ * @author Oleg Nenashev
+ */
+public class EnvInjectJobPropertyTest {
+    
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    public void testVariableInjection() throws Exception {   
+        FreeStyleProject project = j.createFreeStyleProject();
```

Comment:
move such repeated lines to field with init  in`@Before`. (This simple refactoring saves 6 lines here with cost of 4, not bad)